src/webui/webapplication.cpp

```diff
@@ -774,6 +777,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
```

Comment:
>But pointing out changes that make the functionality inconsistent with the rest of the file/project do not make sense to me.

This has nothing to do with style consistency, it's just a better alternative method suggestion.

>If the project wants to change from using split() to splitRef() then that should be done in another PR addressing all uses of it. 

These 2 are not equivalent/interchangeable... the new code here is eligible for it, hence the suggestion.

I'll rest my case here, I'm not the type of insisting/enforcing my opinions...